docs/contributors_guide/maintenance/Ruby_Gem_Release_Procedures.rst

```diff
@@ -0,0 +1,18 @@
+Ruby Gem Release Procedures
+===========================
+
+This section will decribe the release procedures for the SIMP Ruby gems
+used to build and test SIMP components. The relevant components include
+
+* ``rubygem-simp-beaker-helpers``
+* ``rubygem-simp-rake-helpers``
```

Comment:
@opt-ct I didn't see simp-build-helpers referenced anywhere.  If this is a project that we need to release, then we need to add deploy steps into its .travis.yml file.